docs/dev/retool/workflow.md

```diff
@@ -1 +1,74 @@
 # Retool Workflow
+
+[Retool workflows](https://retool.com/products/workflows) are great for automated tasks that need to be run on intervals. For our project, we use Retool workflows to routinely sync our database with Legal Server every day. Additionally, we take advantage of Retool workflows to create function abstractions that chain REST APIs to create, edit, or delete listings (i.e. limited case assignments) via the Retool application.
+
+## Legal Server Sync
+
+To get data from Legal Server, we added the provided API link as a REST API resource.
+After fetching data from Legal Server using said resource, we processed it as follows:
+
+1. Filter to only open cases
+2. Search legal problem code for keywords (i.e. asylum)
+3. Determine time commitment by legal problems
+4. Upsert cases and relevant languages and relief codes to Supabase
+5. Delete old stored cases that do not match what's stored in Legal Server
```

Comment:
gonna drop some wording nits:
- Can we update 1 to: "Filter to only cases seeking volunteers"
- Can we update 3 to: "Determine time commitment based on the case's relief sought value and whether the client is detained"
- 5: should be "LegalServer"